jujugui/static/gui/src/app/init/changes-utils.js

```diff
@@ -282,12 +325,19 @@ const changesUtils = {
               ' ' + machineType +
               (change.command.args[0].length !== 1 ? 's' : '') +
               ' will be destroyed.';
+          changeItem.command = 'juju remove-machine ' +
+            `${change.command.args[0].join(' ')}`;
           break;
         case '_set_config':
           const cfgServ = services.getById(change.command.args[0]);
+          console.log('_set_config', change);
+          ghostService = services.getById(change.command.args[0]);
+          const config = changesUtils._objectToParams(change.command.args[1]);
           changeItem.icon = 'changes-config-changed';
           changeItem.description = 'Configuration values will be changed for ' +
               cfgServ.get('displayName').match(removeBrackets)[1] + '.';
+          changeItem.command = `juju config ${ghostService.get('name')} ` +
+            `${config}`;
           break;
         case '_addKeys':
           changeItem.icon = 'changes-unknown';
```

Comment:
I don't see tests for all these commands: istm that only "juju deploy" is exercised by existing tests.

Reply:
Added.